modules/tinymce/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `editor.insertContent` API would insert contents inside noneditable elements if the selection was inside the element. #TINY-9462
 - Closing a dialog would scroll down the document in Safari. #TINY-9148
 - Quick toolbars were incorrectly rendered during the dragging of `contenteditable="false"` elements. #TINY-9305
+- Selection of images, hrs, tables or noneditable elements was possible if they where within a noneditable root. #TINY-9473
```

Comment:
So we're blocking selection now? Because that's the opposite of the PR title (which will become the commit message) 🙂

Reply:
Good point. Maybe I update the titles here to make it more clear.

Reply:
We are blocking selection inside noneditables including the root so I changed it to a more generic title. I also updated the changelog to reflect that.